src/routes/Header.svelte

```diff
@@ -1,129 +1,86 @@
-</script>
-
-<header>
-  <div class="corner">
-    <a href="https://kit.svelte.dev">
-      <img src={logo} alt="SvelteKit" />
-    </a>
-  </div>
-
-  <nav>
-    <svg viewBox="0 0 2 3" aria-hidden="true">
-      <path d="M0,0 L1,2 C1.5,3 1.5,3 2,3 L2,0 Z" />
-    </svg>
-    <ul>
-      <li aria-current={$page.url.pathname === "/" ? "page" : undefined}>
-        <a href="/">Home</a>
-      </li>
-      <li aria-current={$page.url.pathname === "/courses" ? "page" : undefined}>
-        <a href="/courses">Courses</a>
-      </li>
-      <li aria-current={$page.url.pathname === "/about" ? "page" : undefined}>
-        <a href="/about">About</a>
-      </li>
-    </ul>
-    <svg viewBox="0 0 2 3" aria-hidden="true">
-      <path d="M0,0 L0,3 C0.5,3 0.5,3 1,2 L2,0 Z" />
-    </svg>
-  </nav>
-
-  <div class="corner">
-    <a href="https://github.com/Emmanuel-Melon/planet-of-the-bugs">
-      <img src={github} alt="GitHub" />
-    </a>
-  </div>
-</header>
+// @ts-nocheck
 
-<style>
-  header {
-    display: flex;
-    justify-content: space-between;
-  }
+  import { page } from '$app/stores';
+  import { onMount, onDestroy } from 'svelte';
+  import { signOut } from '@auth/sveltekit/client';
+  import "iconify-icon";
 
-  .corner {
-    width: 3em;
-    height: 3em;
-  }
 
-  .corner a {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 100%;
-    height: 100%;
-  }
+  let user;
+  $: current = $page.url.pathname
+  $: menuLinks = [
+    { text: 'Home', path: '/' },
+    { text: 'Courses', path: '/courses' },
+    { text: 'About', path: '/about' }
+  ];
 
-  .corner img {
-    width: 2em;
-    height: 2em;
-    object-fit: contain;
-  }
+  onMount(() => {
+    user = $page.data.session?.user;
+  });
 
-  nav {
-    display: flex;
-    justify-content: center;
-    --background: rgba(255, 255, 255, 0.7);
-  }
+  onDestroy(() => {
+    user = null;
+    current = null;
+    isMenuOpen = false;
+    menuLinks = [];
+  });
 
-  svg {
-    width: 2em;
-    height: 3em;
-    display: block;
-  }
-
-  path {
-    fill: var(--background);
-  }
-
-  ul {
-    position: relative;
-    padding: 0;
-    margin: 0;
-    height: 3em;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    list-style: none;
-    background: var(--background);
-    background-size: contain;
-  }
+</script>
 
-  li {
-    position: relative;
-    height: 100%;
-  }
+<header class="navbar bg-base-100">
+    <div class="navbar-start">
+      <a href='/' class="btn btn-ghost normal-case text-xl">Planet Of The Bugs</a>
+    </div>
 
-  li[aria-current="page"]::before {
-    --size: 6px;
-    content: "";
-    width: 0;
-    height: 0;
-    position: absolute;
-    top: 0;
-    left: calc(50% - var(--size));
-    border: var(--size) solid transparent;
-    border-top: var(--size) solid var(--color-theme-1);
-  }
+    <ul class="navbar-center hidden lg:flex justify-between items-center w-1/3 text-lg font-bold ml-8">
```

Comment:
LGTM 🚀🚀🚀
BTW, could we replace justify-between with a gap of either 4 or 8? The link spacing is currently too large!

Reply:
ul was taking 1/3rd of the parent. Switched it too 1/5th